multi-node/aws/pkg/config/config.go

```diff
@@ -74,9 +74,27 @@ func ClusterFromBytes(data []byte) (*Cluster, error) {
 	// as it will with RecordSets
 	c.HostedZone = WithTrailingDot(c.HostedZone)
 
+	// If the user specified no subnets, we assume that a single AZ configuration with the default instanceCIDR is demanded
+	if len(c.Subnets) == 0 && c.InstanceCIDR == "" {
+		c.InstanceCIDR = "10.0.0.0/24"
+	}
+
 	if err := c.valid(); err != nil {
 		return nil, fmt.Errorf("invalid cluster: %v", err)
 	}
+
+	// For backward-compatibility
```

Comment:
I think this comment should be more clear what the backwards-compatibility is for, it'll be unclear outside
the context of the commit.  It also implies to me that single AZ deployments are being deprecated, is that going to be the case (cc @colhom)?

Perhaps it should just say something like "fall back to single AZ if no subnets are specified"?